src/System.Reflection.Metadata/src/System/Reflection/Internal/Utilities/FileStreamReadLightUp.cs

```diff
@@ -142,33 +79,16 @@ internal static unsafe bool TryReadFile(Stream stream, byte* buffer, long start,
             return true;
         }
 
-
-// The library guards against unavailable entrypoints by using EntryPointNotFoundException.
-#pragma warning disable BCL0015 // Diasable Pinvoke analyzer errors.
-        private static unsafe class NativeMethods
-        {
-            // API sets available on modern platforms:
-            [DllImport(@"api-ms-win-core-file-l1-1-0.dll", EntryPoint = "ReadFile", ExactSpelling = true, SetLastError = true)]
-            [return: MarshalAs(UnmanagedType.Bool)]
-            internal static extern bool ReadFileModern(
-                 SafeHandle fileHandle,
-                 byte* buffer,
-                 int byteCount,
-                 out int bytesRead,
-                 IntPtr overlapped
-            );
-
-            // older Windows systems:
-            [DllImport(@"kernel32.dll", EntryPoint = "ReadFile", ExactSpelling = true, SetLastError = true)]
-            [return: MarshalAs(UnmanagedType.Bool)]
-            internal static extern bool ReadFileCompat(
-                 SafeHandle fileHandle,
-                 byte* buffer,
-                 int byteCount,
-                 out int bytesRead,
-                 IntPtr overlapped
-            );
-        }
+#pragma warning disable BCL0015 // Disable Pinvoke analyzer errors.
```

Comment:
> For ReadFile in particular it is already in both OneCore and UWP approved list

That's what I thought too. But the build still complained when I have removed it.

> I can do this

Thanks. I do not plan to do anything more for this.